.github/workflows/milestone_adder.yml

```
@@ -0,0 +1,101 @@
+name: PR Milestone Checker
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, edited, ready_for_review, demilestoned]
+  pull_request_target:
+    types: [demilestoned]
```

actionlint (1.7.4)

invalid activity type "demilestoned" for "pull_request_target" Webhook event. available types are "assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "edited", "labeled", "locked", "opened", "ready_for_review", "reopened", "review_request_removed", "review_requested", "synchronize", "unassigned", "unlabeled", "unlocked"

(events)